Index: IzvorniKod/proba.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>proba\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/IzvorniKod/proba.txt b/IzvorniKod/proba.txt
--- a/IzvorniKod/proba.txt	(revision a4ac53f563c87793f5fb8638ef5852a4a0d7a18e)
+++ b/IzvorniKod/proba.txt	(date 1729511409483)
@@ -1,1 +1,0 @@
-proba
